src/app/core/models/issue.model.ts

```diff
@@ -204,6 +204,17 @@ export class Issue {
       `# Items for the Tester to Verify\n${this.getTesterResponsesString(this.testerResponses)}`;
   }
 
+  /**
+   * Gets the number of unresolved disputes in an Issue.
+   */
+  getUnresolvedDisputes(): number {
```

Comment:
This name is quite misleading. How about `numOfUnresolvedDisputes`?

Reply:
Agreed and fix